README.rst

```diff
@@ -31,7 +31,7 @@ It is meant to be used as the technical content to support the `Aiven's devporta
       - `Go <postgresql/go/README.md>`__, `Python <postgresql/python/README.md>`__
     * - **OpenSearch**
       - `Python <opensearch/python/README.md>`__  
-    * - **Elasticsearch**
+    * - **ElasticSearch**
```

Comment:
Shouldn't we keep `Elasticsearch` like it is mentioned  at https://www.elastic.co/elasticsearch/